env/production/config.json

```diff
@@ -110,6 +110,6 @@
   "OIDC_GROUPS_CLAIM": "cognito:groups",
   "SESSION_COOKIE_DOMAIN": "nextstrain.org",
   "GROUPS_DATA_FILE": "groups.json",
-  "RESOURCE_INDEX": "s3://nextstrain-inventories/resources/v6.json.gz",
+  "RESOURCE_INDEX": "s3://nextstrain-inventories/resources/v7.json.gz",
```

Comment:
I don't think this needs an update, but also fine to update it...

Reply:
Yeah, not totally necessary, but I didn't want this change propagating to prod before we promote.